content/en/cd-as-a-service/concepts/deployment/traffic-management/smi-linkerd.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Traffic Management Using Linkerd
```

Comment:
I will propose title change to Traffic Management Using SMI, same for linkTitle.

We support SMI which is a standard and LinkerD complies to that standard, there are few other service meshes that also comply with SMI standard.

Reply:
I changed the title from the generic SMI because CD-as-a-Service doesn't support SMI-based service meshes other than Linkerd.  I thought it was kinda weird to talk about SMI service meshes but then state that we only support using Linkerd. I planned on changing the title if/when we support more than Linkerd. thoughts?